### DISCUSSION TITLE
fresh install questions

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [OpenSurv/OpenSurv#26] [General]
I have a fresh install of Ubuntu 24.04 on a Beelink U59 (Intel N5105 with 16 GB ram). I've run the installer script for OpenSurv. It asked me whether I wanted to overwrite the config and then whether I wanted restart opensurv after install. I answered yes to both questions. Then it said I needed to reboot because it was my first install. When I pressed ```<enter>``` to reboot, I got this error:

```
Failed to restart lightdm.service: Transaction for lightdm.service/restart is destructive (blk-availability.service has 'stop' job queued, but 'start' is included in transaction).
See system logs and 'systemctl status lightdm.service' for details.
```

Is this to be expected?

Just to be clear, when it reboots, it does successfully reboot into OpenSurv and display the demo images/video. I just want to be sure that the error message described above does not indicate some kind of problem.

**Comment by SvenVD** [comment]
If OpenSurv starts fine after reboot then everything is fine.
